app/sass/_layout.scss

```diff
@@ -0,0 +1,294 @@
+/*
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+*/
+
+/**
+ * There styles were pulled from iron-flex-layout/iron-flex-layout-classes.html
+ * so that they are in the main stylesheet.
```

Comment:
No, there are still other elements (e.g. pw-shell, demo-tabs, pw-footer) that rely on those classes (and because they're in shadow roots, they don't share the same stylesheet).